standalone-tools/wss-scan.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+###### Information ############################################################################
+# Name:          wss-scan.sh
+# Maintainer:    ODFE Infra Team
+# Language:      Shell
+#
+# About:         This script is to scan the odfe distros for vulnerabilities and licenses
+#                It will scan the repositories and send the WhiteSource link to the mail 
+#                of the user. 
+#
+# Prerequisites: Need to install Java 11
+#                Export JAVA_HOME env variable to the JDK path
+#		 Add JAVA_HOME to PATH variable
+#                Need to set the recepient mail in wss-scan.config for local run
+#                WhiteSource API key is needed for local run, The API Key can be retrieved from the
+#                WhiteSource Admin Console of your account. Use the below command to export the API key
+#                export wss_apikey=$(APIKEY)
+#
+# Usage:         ./wss-scan.sh
+#
+# Starting Date: 10-13-2020
+# Modified Date: 10-21-2020
```

Comment:
Remove. These add no value. The git commit history has all this information and more.